styles.css

```diff
@@ -1 +1,66 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+.container {
+        display: grid;
+        grid-auto-columns: 1fr;
+        grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+        gap: 16px;
+                grid-template-areas:
+                "blocka"
+                "blockb"
+                "blockc"
+                "blockd"
+                "blocke"
+                "."
+                "blockf";
+}
+
+/*desktopgrid*/
+
+@media screen and (min-width:576px) {
```

Comment:
Normaal gezien gaan media queries in een apart bestand. Maar in dit geval zou ik of je media querie bovenaan of onderaan zetten.